8ball/assetlist.json

```diff
@@ -0,0 +1,28 @@
+{
+    "$schema": "../assetlist.schema.json",
+    "chain_name": "8ball",
+    "assets": [
+      {
+        "description": "The native staking token of 8ball.",
+        "denom_units": [
+          {
+            "denom": "uebl",
+            "exponent": 0
+          },
+          {
+            "denom": "ebl",
+            "exponent": 6
+          }
+        ],
+        "base": "uebl",
+        "name": "8ball",
+        "display": "8ball",
+        "symbol": "EBL",
+        "logo_URIs": {
+          "png": "https://8ball.info/8ball.png",
```

Comment:
oh, these URIs need the same fix (must point to this repo)

Reply:
Fixed. Thank you